enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -189,7 +203,7 @@
             setting( "dbms.security.ldap.authorization.system_password", STRING, NO_DEFAULT );
 
     @Description( "The name of the base object or named context to search for user objects when LDAP authorization is " +
-                  "enabled." )
+                  "enabled. A common case is that this matches the last part of `dbms.security.ldap.user_dn_template`." )
```

Comment:
This is also wrong in two places in the comments in neo4j.conf. Maybe you could fix it in this PR?